cyto_dl/api/data.py

```diff
@@ -0,0 +1,43 @@
+# goal: allow the API client to interact with CytoDLModel without
+# worrying about the actual values being used in config files
+from enum import Enum
+
+import skimage
+
+
+class ExperimentType(Enum):
+    GAN = "gan"
+    INSTANCE_SEG = "instance_seg"
+    LABEL_FREE = "labelfree"
+    SEGMENTATION_PLUGIN = "segmentation_plugin"
+    SEGMENTATION = "segmentation"
+
+
+class HardwareType(Enum):
+    CPU = "cpu"
+    GPU = "gpu"
+    # other hardware types available, but require more complicated config
+
+
+class PatchSize(Enum):
+    """Patch size for training, and their respective patch shapes.
+
+    TODO: get from benji
+    """
+
+    SMALL = [1, 3, 3]
```

Comment:
Brian and I talked about updating these to 
small = [16, 32, 32]
medium = [16, 64, 64]
large = [16, 128, 128]

Reply:
Done ([commmit](https://github.com/AllenCellModeling/cyto-dl/pull/347/commits/0e3828a151966ff1ae2550a4db25cdf285382d02))